bots/twitch/sogebot/README.md

```diff
@@ -1,7 +1,6 @@
 # SogeBot
 ### Their [Site](https://www.sogebot.xyz)
-sogBot is an actively developed open source interactive Twitch bot that provides entertainment and moderation for your channel,
-allowing you to focus on what matters the most to you - your game and your viewers.
+SogeBot is an actively developed open source interactive Twitch bot that provides entertainment and moderation for your channel, allowing you to focus on what matters the most to you - your game and your viewers.
```

Comment:
This is broken like this to be readable in a text editor instead of one long line.

Markdown treats these the same, while one is much less readable.